test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
 require 'minitest/autorun'
-$:.unshift lib_dir unless $:.include?(lib_dir)
-require 'debug_inspector'
+Bundler.require :default
```

Comment:
I think we do not have set `Bundler.require :default`.
Maybe it works without this line.